rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -1230,13 +1235,15 @@ private Transaction buildUpdateTx() {
     }
 
     private BridgeSupport initBridgeSupport(BridgeEventLogger eventLogger, ActivationConfig.ForBlock activationMock) {
+        when(feePerKbSupport.getFeePerKb()).thenReturn(bridgeConstants.getFeePerKbConstants().getGenesisFeePerKb());
```

Comment:
Let's move this to `setUpOnEachTest` method and put it right after the instantiation of `feePerKbSupport` field. Wdyt?

Reply:
Done!